borg/testsuite/archiver.py

```diff
@@ -274,7 +272,8 @@ def create_test_files(self):
             # same for newer ubuntu and centos.
             # if this is supported just on specific platform, platform should be checked first,
             # so that the test setup for all tests using it does not fail here always for others.
-            # xattr.setxattr(os.path.join(self.input_path, 'link1'), 'user.foo_symlink', b'bar_symlink', follow_symlinks=False)
+            # xattr.setxattr(os.path.join(self.input_path, 'link1'), 'user.foo_symlink', b'bar_symlink',
+            # follow_symlinks=False)
```

Comment:
maybe stay away from reformatting the src while doing functional changes.
also, 80 < line length < 120 is pretty much ok for borg.